Openshift-EE/pipelines/OpenEBS-base/stages/2-setup/1CXH-storage-policies/openebs-sc

```diff
@@ -67,4 +67,4 @@ if [ "$1" == "node" ];then
   node $2 $3 $4
 else
   pod
-fi
```

Comment:
@w3aman there is no change in this file ?

Reply:
yes there is no change.